go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cosmos/gaia/v5
 
-go 1.16
+go 1.17
```

Comment:
There is no need for this bump Jacob. Please let the user build with whichever version of Go > 1.16 they think it's best.

Reply:
You're right that there's no explicit need for it. 

The reasoning is that the go1.17 modules are clearer and easier to read.

Reply:
maybe I can have the go1.17 formatting and go1.16 compatibility?  

I will check on that.

Reply:
The only possibly significant issue is that the SDK v0.42 is not being built and tested with go 1.17